internal/pkg/cli/env_init.go

```diff
@@ -429,8 +429,15 @@ func (o *initEnvOpts) askEnvSession() error {
 
 	selCreds, err := o.selCreds()
 	if err != nil {
-		return err
+		errRetrieveCreds := err
+		sess, err := o.sessProvider.Default()
```

Comment:
so if we fallback to the default credential, how do we set `selCreds` to make sure

```golang
sess, err := selCreds.Creds(fmt.Sprintf(fmtEnvInitCredsPrompt, color.HighlightUserInput(o.name)), envInitCredsHelpPrompt)
```

this works?

Reply:
We don't. When it falls back it returns before then. That's because that line of code is specifically what we're avoiding calling.

Reply:
oh sorry i misread that.